.travis.yml

```diff
@@ -0,0 +1,196 @@
+      addons:
+        apt:
+          sources:
+          - llvm-toolchain-precise-3.5
+          # needed for libstdc++ headers
+          - ubuntu-toolchain-r-test
+          packages:
+          - clang-3.5
+          - libstdc++6-4.6-dbg
+          - libnl-3-200
+          - libnl-3-dev
+          - libnl-route-3-200
+          - libnl-route-3-dev
+          - libibverbs-dev
+          - librdmacm-dev
+    - compiler: clang
+      env: USED_CC=clang-3.6     USED_CXX=clang++-3.6
+      addons:
+        apt:
+          sources:
+          - llvm-toolchain-precise-3.6
+          # needed for libstdc++ headers
+          - ubuntu-toolchain-r-test
+          packages:
+          - clang-3.6
+          - libstdc++6-4.6-dbg
+          - libnl-3-200
+          - libnl-3-dev
+          - libnl-route-3-200
+          - libnl-route-3-dev
+          - libibverbs-dev
+          - librdmacm-dev
+    - compiler: clang
+      env: USED_CC=clang-3.7     USED_CXX=clang++-3.7
+      addons:
+        apt:
+          sources:
+          - llvm-toolchain-precise-3.7
+          # needed for libstdc++ headers
+          - ubuntu-toolchain-r-test
+          packages:
+          - clang-3.7
+          - libstdc++6-4.6-dbg
+          - libnl-3-200
+          - libnl-3-dev
+          - libnl-route-3-200
+          - libnl-route-3-dev
+          - libibverbs-dev
+          - librdmacm-dev
+    - compiler: gcc
+      env: USED_CC=gcc-4.4     USED_CXX=g++-4.4
+      addons:
+        apt:
+          sources:
+          - ubuntu-toolchain-r-test
+          packages:
+          - g++-4.4
+          - libstdc++6-4.4-dbg
+          - libnl-3-200
+          - libnl-3-dev
+          - libnl-route-3-200
+          - libnl-route-3-dev
+          - libibverbs-dev
+          - librdmacm-dev
+    - compiler: gcc
+      env: USED_CC=gcc-4.7     USED_CXX=g++-4.7
+      addons:
+        apt:
+          sources:
+          - ubuntu-toolchain-r-test
+          packages:
+          - g++-4.7
+          - libstdc++6-4.7-dbg
+          - libnl-3-200
+          - libnl-3-dev
+          - libnl-route-3-200
+          - libnl-route-3-dev
+          - libibverbs-dev
+          - librdmacm-dev
+    - compiler: gcc
+      env: USED_CC=gcc-4.8     USED_CXX=g++-4.8
+      addons:
+        apt:
+          sources:
+          - ubuntu-toolchain-r-test
+          packages:
+          - g++-4.8
+          - libstdc++6-4.8-dbg
+          - libnl-3-200
+          - libnl-3-dev
+          - libnl-route-3-200
+          - libnl-route-3-dev
+          - libibverbs-dev
+          - librdmacm-dev
+    - compiler: gcc
+      env: USED_CC=gcc-4.9     USED_CXX=g++-4.9
+      addons:
+        apt:
+          sources:
+          - ubuntu-toolchain-r-test
+          packages:
+          - g++-4.9
+          - libstdc++6-4.9-dbg
+          - libnl-3-200
+          - libnl-3-dev
+          - libnl-route-3-200
+          - libnl-route-3-dev
+          - libibverbs-dev
+          - librdmacm-dev
+    - compiler: gcc
```

Comment:
Can you add env gcc-6?
Gcc-6 is used in fedora 24 where libvma is an official package